edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Edgelet.Test/ModuleIdentityLifecycleManagerTest.cs

```diff
@@ -24,11 +24,11 @@ public class ModuleIdentityLifecycleManagerTest
         static readonly ModuleIdentityProviderServiceBuilder ModuleIdentityProviderServiceBuilder = new ModuleIdentityProviderServiceBuilder(IothubHostName, DeviceId);
 
         [Fact]
-        public async Task TestGetModulesIdentity_WithEmptyDiff_ShouldReturnEmptyIdentities()
+        public async Task TestGetModulesIdentity_WithEmptyDiffAndEmptyCurrent_ShouldReturnEmptyIdentities()
         {
             // Arrange
             var identityManager = Mock.Of<IIdentityManager>(m => m.GetIdentities() == Task.FromResult(Enumerable.Empty<Identity>()));
-            var moduleIdentityLifecycleManager = new ModuleIdentityLifecycleManager(identityManager, ModuleIdentityProviderServiceBuilder, EdgeletUri);
+            var moduleIdentityLifecycleManager = new ModuleIdentityLifecycleManager(identityManager, ModuleIdentityProviderServiceBuilder, EdgeletUri, false);
```

Comment:
maybe it should test with both false and true to make sure it doesn't brake existing cases when flag is on